Python/import.c

```diff
@@ -905,26 +905,29 @@ PyImport_AddModule(const char *name)
 }
 
 
-/* Remove name from sys.modules, if it's there. */
+/* Remove name from sys.modules, if it's there.
+ * Can be called with an exception raised.
+ * If fail to remove name a new exception will be chained with the old
```

Comment:
```suggestion
 * If we fail to remove 'name' then a new exception will be chained with the old
```

Reply:
Thank you, but this is a backport, and different comment was committed in master. I think it is better to fix it in master first and backport to 3.9.